_projects/2023/100483841/100483841.Rmd

```diff
@@ -0,0 +1,238 @@
+---
+title: "Population Perspectives"
+description: |
+  A graph about population in 2017 and 2100 will be replicated and an alternative way of visualizing this graph will be presented
+author: Victoria Costas Salamanca
+date = date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+```{r}
+#Libraries needed
+library(tidyverse)
+library(ggalluvial)
+library(ggplot2)
+library(ggrepel)
+library(extrafont)
+library(showtext)
+library(ggtext)
+library(htmlwidgets)
+library(ggbump)
+```
+
+## Dataframe
+
+
+```{r}
+#Create my database:
+
+#To use with the population
+B = 10^9 #Billion
+M = 10^6 #Million
+
+year <- rep(c(2017, 2100), 15)
+population <- c(1.4 * B, 732 * M, 1.38 * B, 1.09 * B, 325 * M, 336 * M, 258 * M,
+                229 * M, 214 * M, 248 * M, 212 * M, 165 * M, 206 * M, 791 * M, 
+                157 * M, 81 * M, 146 * M, 106 * M, 128 * M, 60 * M, 112*M, 170*M, 103 * M,
+                223 * M, 96 * M, 199 * M, 81 * M, 246 * M, 54 * M, 186 * M)
+country <- c("China", "China","India", "India", "U.S", "U.S",
+             "Indonesia", "Indonesia", "Pakistan", "Pakistan",
+             "Brazil", "Brazil", "Nigeria", "Nigeria", "Bangladesh", 
+             "Bangladesh", "Russia", "Russia", "Japan", "Japan", "VIRTUAL", "VIRTUAL",
+             "Ethiopia", "Ethiopia", "Egypt", "Egypt", "DRC", "DRC",
+             "Tanzania", "Tanzania")
+#One virtual country was created in orther to have white space between countries as in the original graph
+group_colors <- c(China = "#009ee5", India = "#0973cb", U.S = "#384aae",
+                Indonesia = "#603c9c", Pakistan = "#893e9b", Brazil ="#be388f", 
+                Nigeria = "#e75c93", Bangladesh = "#e53e48", Russia = "#f96439",
+                Japan = "#f78c24", VIRTUAL = "#FFFFFF", Ethiopia = "#f5b314", Egypt = "#98b245", 
+                DRC ="#40b87b", Tanzania = "#25a99b")
+ranking <- c( 15, 13, 14, 15, 13, 12, 12, 9, 11, 11, 10, 4, 9, 14, 8, 2, 7, 3, 6, 1,
+              5, 5, 4, 8, 3, 7, 2, 10, 1, 6)
+
+#Create dataframe
+df <- data.frame(x = year,
```

Comment:
It would be better to save this in a CSV, include this CSV next to the Rmd, and then just call here `df <- read.csv("database.csv")`, and you can remove this whole chunk.